netbout-client/src/test/java/com/netbout/client/RtAliasITCase.java

```diff
@@ -38,6 +38,7 @@
  * Integration case for {@link RtAlias}.
  * @author Matteo Barbieri (barbieri.matteo@gmail.com)
  * @version $Id$
+ * @since 2.0
```

Comment:
@kujtimiihoxha it was implemented in e49870f0bcb92d8db6a0a719db7656af7c126c67#diff-b986fdcc876b9b4d8b0882e9bfb32588 for `2.15` release